modal.js

```diff
@@ -0,0 +1,247 @@
+const body2 = document.querySelector('body');
+const aboutMe = document.querySelector('.about-me');
+const cardContainer = document.createElement('section');
+cardContainer.classList.add('card-works');
+const buttonsArr = [];
+const cardArr = [];
+const modalArr = [];
+const closeModalArr = [];
+
+const projects = [
+  {
+    name: 'Tonic',
+    featuredImage: "url('./images/Snapshoot-Portfolio-1.png')",
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    languagesList: ['html', 'css', 'javascript'],
+    liveLink: 'https://fikerte-t.github.io/Microverse-Student-Project-1/',
+    sourceLink: 'https://github.com/Fikerte-T/Microverse-Student-Project-1/',
+  },
+  {
+    name: 'Multi-Post Stories',
+    featuredImage: "url('./images/Snapshoot-Portfolio-2.png')",
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    languagesList: ['html', 'Ruby on rails', 'css', 'javascript'],
+    liveLink: 'https://fikerte-t.github.io/Microverse-Student-Project-1/',
+    sourceLink: 'https://github.com/Fikerte-T/Microverse-Student-Project-1/',
+  },
+  {
+    name: 'Facebook 360',
+    featuredImage: "url('./images/Snapshoot-Portfolio-3.png')",
+    description: 'Exploring the future of media in Facebook\'s first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    languagesList: ['html', 'Ruby on rails', 'css', 'javascript'],
+    liveLink: 'https://fikerte-t.github.io/Microverse-Student-Project-1/',
+    sourceLink: 'https://github.com/Fikerte-T/Microverse-Student-Project-1/',
+  },
+  {
+    name: 'Uber Navigation',
+    featuredImage: "url('./images/Snapshoot-Portfolio-4.png')",
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    languagesList: ['html', 'Ruby on rails', 'css', 'javascript'],
+    liveLink: 'https://fikerte-t.github.io/Microverse-Student-Project-1/',
+    sourceLink: 'https://github.com/Fikerte-T/Microverse-Student-Project-1/',
+  },
+];
+
+projects.forEach((project) => {
+  body2.appendChild(cardContainer);
+  document.body.insertBefore(cardContainer, aboutMe);
+  const card = document.createElement('div');
+  card.classList.add('card');
+  cardContainer.appendChild(card);
```

Comment:
- [ ] While you were implementing the popup feature, you did not entirely put the page's responsiveness into consideration. Adjust and ensure that both mobile and desktop versions are responsive.
![Screenshot from 2021-09-02 11-58-10](https://user-images.githubusercontent.com/65501535/131815719-a1578d36-5246-4479-bbff-5d84f9d2ca80.png)
![Screenshot from 2021-09-02 12-05-26](https://user-images.githubusercontent.com/65501535/131815994-8b8d1de9-5dc9-479c-ab8d-97f246473c43.png)